springfield/firefox/templates/firefox/all/mobile.html

```diff
@@ -16,7 +16,7 @@
   {% set qr_url = android_nightly_url %}
   {% set android_url = android_nightly_url %}
 {% else %}
-  {% set qr_url ="https://www.firefox.com/firefox/browsers/mobile/app/?product=firefox&campaign=firefox-all" %}
+  {% set qr_url ="https://www.firefox.com/browsers/mobile/app/?product=firefox&campaign=firefox-all" %}
```

Comment:
Meaningless OCD whitespace while touching the line:

```suggestion
  {% set qr_url = "https://www.firefox.com/browsers/mobile/app/?product=firefox&campaign=firefox-all" %}
```